help.txt

```diff
@@ -56,7 +56,7 @@ asdf env <command> [util]               Runs util (default: `env`) inside the
 asdf info                               Print OS, Shell and ASDF debug information.
 asdf version                            Print the currently installed version of ASDF
 asdf reshim <name> <version>            Recreate shims for version of a package
-asdf shim-versions <command>            List the plugins and versions that
+asdf shimversions <command>             List the plugins and versions that
```

Comment:
This file isn't actually read by the Go code, it only exists because the Bash code which still exists in the repo reads it. Can you revert this change from just this file?

Reply:
It is used by the docs at https://github.com/asdf-vm/asdf/blob/9502b564b0a986a9edf9b9dd37565f2220c5d901/docs/manage/commands.md?plain=1#L5

Reply:
This is fixed in https://github.com/asdf-vm/asdf/pull/1890